### PR TITLE
react-multiplatform-app: hardwares

### PR DESCRIPTION
**Pull request by Kompanel**

### CODE REVIEW

File: react-multiplatform-app/app.json [hunk 1]
```diff
@@ -15,13 +15,23 @@
...
     },
     "android": {
       "adaptiveIcon": {
         "foregroundImage": "./assets/adaptive-icon.png",
         "backgroundColor": "#ffffff"
-      }
+      },
+      "permissions": [
+        "android.permission.CAMERA",
+        "android.permission.RECORD_AUDIO"
```
Comment (Crysisek) [inline react-multiplatform-app/app.json]: Record audio permission is necessary?

File: react-multiplatform-app/src/pages/Animation.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+});
+
+export default Animation
```
Comment (Crysisek) [inline react-multiplatform-app/src/pages/Animation.tsx:19]: File not ending with empty line. Is it ok in react project?

File: react-multiplatform-app/src/pages/Hardware.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+const Hardware = () => {
+  const navigation = useNavigation();
+
+  const buttons = [
+    { id: 10, title: 'Accelerometer', routeName: 'Accelerometer' },
+    { id: 20, title: 'Camera', routeName: 'Camera' },
+    { id: 30, title: 'Gps', routeName: 'Gps' },
+    { id: 40, title: 'Light', routeName: 'Light' },
+    { id: 50, title: 'Microphone', routeName: 'Microphone' },
```
Comment (Crysisek) [inline react-multiplatform-app/src/pages/Hardware.tsx]: I think in android I have made additional category here, named sensors and inside I put light and Accelerometer.

Comment (Crysisek) [inline react-multiplatform-app/src/pages/Hardware.tsx]: + are we adding microphone?

File: react-multiplatform-app/src/pages/Animation.tsx (hunk 2 above)
Comment (Kompanel) [reply react-multiplatform-app/src/pages/Animation.tsx:19]: Not necessary (eslint disabled)

File: react-multiplatform-app/app.json (hunk 1 above)
Comment (Kompanel) [reply react-multiplatform-app/app.json]: Fixed

File: react-multiplatform-app/src/pages/Hardware.tsx (hunk 3 above)
Comment (Kompanel) [reply react-multiplatform-app/src/pages/Hardware.tsx]: Sensors are named "hardware" in my app, will change

**Review by Crysisek — APPROVED** [review]

**Pull request merged**